src/applications/disability-benefits/all-claims/content/mentalHealth.jsx

```diff
@@ -0,0 +1,186 @@
+  'Examples of mental health disorders include, but are not limited to, post-traumatic stress disorder (PTSD), depression, anxiety, and bipolar disorder.';
+
+export const traumaticEventsInfo = (
+  <va-accordion open-single>
+    <va-accordion-item
+      header="Examples of traumatic events"
+      class="vads-u-margin-y--3"
+      id="first"
+      bordered
+    >
+      <h4 className="vads-u-margin-top--0">
+        Traumatic events related to combat
+      </h4>
+      <ul>
+        <li>You were engaged in combat with enemy forces</li>
+        <li>You experienced fear of hostile military or terrorist activity</li>
+        <li>You served in an imminent danger area</li>
+        <li>You served as a drone aircraft crew member</li>
+      </ul>
+      <h4 className="vads-u-margin-top--0">
+        Traumatic events related to sexual assault or harassment
+      </h4>
+      <ul>
+        <li>
+          You experienced pressure to engage in sexual activities (for example,
+          someone threatened you with bad treatment for refusing sex, or
+          promised you better treatment in exchange for sex)
+        </li>
+        <li>
+          You were pressured into sexual activities against your will (for
+          example, when you were asleep or intoxicated)
+        </li>
+        <li>You were physically forced into sexual activities</li>
+        <li>
+          You experienced offensive comments about your body or sexual
+          activities
+        </li>
+        <li>You experienced unwanted sexual advances</li>
+        <li>
+          You experienced someone touching or grabbing you against your will,
+          including during hazing
+        </li>
+      </ul>
+      <h4 className="vads-u-margin-top--0">
+        Traumatic events related to other personal interactions
+      </h4>
+      <ul>
+        <li>
+          You experienced physical assault, battery, robbery, mugging, stalking,
+          or harassment by a person who wasn’t part of an enemy force
+        </li>
+        <li>You experienced domestic intimate partner abuse or harassment</li>
+      </ul>
+      <h4 className="vads-u-margin-top--0">Other traumatic events</h4>
+      <ul>
+        <li>You got into a car accident</li>
+        <li>You witnessed a natural disaster, like a hurricane</li>
+        <li>You worked on burn ward or graves registration</li>
+        <li>
+          You witnessed the death, injury, or threat to another person or to
+          yourself, that was caused by something other than a hostile military
+          or terrorist activity
+        </li>
+        <li>
+          You experienced or witnessed friendly fire that occurred on a gunnery
+          range during a training mission
+        </li>
+      </ul>
+    </va-accordion-item>
+  </va-accordion>
+);
+
+export const noneAndConditionError =
+  "If you're not claiming any mental health conditions related to a traumatic event, unselect the other options you selected";
```

Comment:
use [curly apostrophe](https://design.va.gov/content-style-guide/punctuation#apostrophes)

```suggestion
 'If you’re not claiming any mental health conditions related to a traumatic event, unselect the other options you selected';
```

Reply:
Corrected in [6eba54f](https://github.com/department-of-veterans-affairs/vets-website/pull/32993/commits/6eba54f875309c70a4cce8918c3b97a6e4b710b6)

<img width="538" alt="Screenshot 2024-11-19 at 11 16 39 AM" src="https://github.com/user-attachments/assets/1910a4e0-61b1-4f43-8425-6139e5234dd3">